pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/executor/OpChainSchedulerService.java

```diff
@@ -117,12 +123,26 @@ public void runJob() {
    */
   public final void register(OpChain operatorChain) {
     register(operatorChain, true);
+    LOGGER.debug("({}): Scheduler is now handling operator chain listening to mailboxes {}. "
```

Comment:
Can this be info level or will that be too noisy ?

Reply:
I think it's likely to be too noisy - `it's O(Num_queries * num_stages_per_query)`. I separated it from the trace calls though so that if you wanted to you could turn on only the once-per-opchain logs